app/imports/components/Sprints/SingleSprint/SprintToolbar.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+import {
+    RaisedButton,
+    Toolbar,
+    ToolbarGroup,
+} from 'material-ui';
+
+const SprintToolbar = ({
+sprint,
+closeSprint,
+isModerator,
+}) =>
+    <Toolbar>
+        <ToolbarGroup>
+            {sprint.name} {sprint.closed ? ' - closed' : ''}
+        </ToolbarGroup>
+
+        {isModerator ? <ToolbarGroup>
+            <RaisedButton
+                label={!sprint.closed ? 'Close sprint' : 'Open sprint'}
```

Comment:
For now change `Open sprint` to `Reopen sprint`, because by default new sprint is open.